### PR TITLE
Remove animations of last label and adding start animation of indicat…

### LINKED ISSUE #14080: [GlobalStep] [iOS] Under "Menus" black empty box flashes for few seconds before "Nothing Found" message is prompted while adding new item.
Environment:
iOS: iPad Air 2(v13.3.1), iPhone 11 Pro (v13.3.1)

Steps To Reproduce:
 
 1. Install and launch the application.
 2. Tap on "Log In".
 3. Log in to the app using valid credentials.
 4. Open "My site" tab.
 5. Tap on "Menus".
 6. Select or add new menu.
 7. Tap on New item and list will open.
 8. Tap on any option from the list (eg. Page,Link, Tag, Post,Product)
 9. Observe that   black empty text box is displayed for few second and then "Nothing found" message is displayed.
  
 Actual Result:
Under "Menus" black empty box flashes for few seconds before "Nothing Found" message is prompted while adding new item

 Expected Result:
Upon adding new item in Menu from "Menus" blank empty box should not display.

[#14080.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/4594795/14080.zip)

### PR DESCRIPTION
**Pull request by negna1**
Fixes #14080 

To test:
1. Open "My site" tab.
2. Tap on "Menus".
3. Select or add new menu.
4. Tap on New item and list will open.
5. Tap on any option from the list (eg. Page,Link, Tag, Post,Product)
6. If this page is loading there must be indicator loading , and when page load it must stop animating.

PR submission checklist:

- [X] I have considered adding unit tests where possible.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Menus/MenuItemSourceFooterView.m [hunk 1]
```diff
@@ -57,8 +58,16 @@ - (void)setupSourceCell
...
     [self addSubview:cell];
+    _spinner = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+    UIWindow *window = [[UIApplication sharedApplication] keyWindow];
+    UIView *topView = window.rootViewController.view;
+    CGFloat xPlace = topView.frame.size.width;
+    CGFloat yPlace = cell.frame.size.height;
+    CGFloat heightOfIndicator = cell.frame.size.height - 4;
+    CGFloat widthOfIndicator = heightOfIndicator;
+    _spinner.frame = CGRectMake(xPlace / 2 - widthOfIndicator / 2 , yPlace / 2 - heightOfIndicator / 2, widthOfIndicator, heightOfIndicator);
+    [self addSubview:_spinner];
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Menus/MenuItemSourceFooterView.m:70]: Could we add this setup to its own method?

Another suggestion:
We could use Autolayout to center the spinner to its superview. In this way we can simplify this snippet :)

**Review by etoledom — COMMENTED** [review]
@negna1 - Thank you so much for this PR, and sorry for taking so long on reviewing it 🙏 

Since we are getting rid of the pulsing animation, I think we could delete all the related code.
For example:
`PulseAnimationDuration`, `MenuItemSourceLoadingDrawView`, `MenuItemSourceLoadingDrawViewDelegate`, the timers, and anything else used for the pulse animation. This would be a pretty nice clean up ;)

Another couple of things I noticed are:

- The activity indicator looks dark on Dark Mode, probably we would like to use StyleMedium on iOS13 and above you avoid this problem.
- The label `Nothing Found` is still present when loading data (after the first load).

IMG_6789E88180A5-1

I also left a code comment with some suggestions.

Please let us know if you would be able to make the requested update 🙏 
Thanks!

**Comment by etoledom** [comment]
It would be great if you could also update this branch from `develop`. Thanks!

**Comment by chipsnyder** [comment]
Hi @negna1 Thank you for putting this fix together. I wanted to check in and see what the progress was on this? If I can assist in any way please let me know :)

**Comment by mindgraffiti** [comment]
Hey there @negna1 👋 

I have incorporated your changes and some additional changes in PR #15214, which was merged in today. Thank you for your work on this!

**Pull request closed without merging**